### ISSUE TITLE
[FR] Push `dist/gcp/*.sha256` files to public repo when building releases for GCP

### ISSUE THREAD

**Issue by roscoejp** [privacysandbox/bidding-auction-servers#11]
It looks like the [./production/packaging/build_and_test_all_in_docker](https://github.com/privacysandbox/bidding-auction-servers/blob/main/production/packaging/build_and_test_all_in_docker) script creates artifacts in the `dist/gcp` directory that includes `*.sha256` files for the binary hashes of a particular build.

If these hash files were uploaded to the repo when a public release is created, they could be used for direct comparison against local/operator built images. This would make public validation of Google's provided hashes a bit easier compared to the current method of scraping the release notes.

**Comment by dankocoj-google** [comment]
Good idea Roscoe, I'll work on this. We'll probably publish artifacts in a format like
`<service_name>.<aws|gcp>.hash`
